bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -403,6 +434,32 @@ public boolean tryRestoreLogin(@Nullable String data, @Nullable String overloade
         return loginTime;
     }
 
+    private @Nullable Authentication tryGetBootstrap() throws IOException, URISyntaxException {
+        HttpsURLConnection connection = makeRequest("GET", alexaServer + "/api/bootstrap", null, false, false, null, 0);
+        String contentType = connection.getContentType();
+        if (connection.getResponseCode() == 200 && StringUtils.startsWithIgnoreCase(contentType, "application/json")) {
+            try
+            {
+                String bootstrapResultJson = convertStream(connection);
+                JsonBootstrapResult result = parseJson(bootstrapResultJson, JsonBootstrapResult.class);
+                Authentication authentication = result.authentication;
+                if (authentication != null && authentication.authenticated) {
+                    this.customerName = authentication.customerName;
+                    if (this.accountCustomerId == null) {
+                        this.accountCustomerId = authentication.customerId;
+                    }
+                    return authentication;
+                }
+            }
```

Comment:
I do not why, but since I have updated my IDE to the latest openHAB structure the auto-formatting feature after saving file in lost. 
But anyway, I fixed it here.

Reply:
I don‘t know why this happens, but you have to select the formatter profile again. Somewhere in the settings you can find java/codestyle/formatter (or similar). ESH is selected, select something else, apply, re-select ESH, apply. Voila, the formatter works again.

Reply:
Thank you for that hint, it is working now!